models/tmp/stg_linkedin_ads__account_history_tmp.sql

```diff
@@ -1,4 +1,14 @@
 {{ config(enabled=var('ad_reporting__linkedin_ads_enabled', True)) }}
 
-select *
-from {{ var('account_history') }}
\ No newline at end of file
+{{
+    fivetran_utils.union_data(
+        table_identifier='account_history', 
+        database_variable='linkedin_database', 
+        schema_variable='linkedin_schema', 
+        default_database=target.database,
+        default_schema='linkedin',
```

Comment:
Correct me if I am wrong, but I think if we want to avoid the fivetran_utils exception then we should have this be `linkedin_ads`. If that is the case then can we apply these updates throughout the package to avoid the need for updating fivetran_utils.

Reply:
Same comment for other tmp models.

Reply:
Updated in line with `linked_ads` instead of `linkedin`.